library/IvozProvider/Mapper/Sql/FaxesInOut.php

```diff
@@ -31,9 +31,8 @@ protected function _save(\IvozProvider\Model\Raw\FaxesInOut $model,
         $statusHaschanged = $model->hasChange("status");
 
         if ($isOutgoingFax && $statusHaschanged && $isPending) {
-            $faxSenderJob = new \IvozProvider\Gearmand\Jobs\FaxSender();
-            $faxSenderJob->setFaxInOut($model);
-            $faxSenderJob->send();
+            $ari = new \Asterisk\ARI\Connector();
+            $ari->sendFaxfileRequest($model);
```

Comment:
The i18n for the mappers errors is something I don't care too much. I see them more related with the portal than mappers.

The code in the other hand is what someone using the API (that will end in the same mapper) will take into account to check what has gone wrong.

Any suggestion for the error code number?